src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/XsltCompileContext.cs

```diff
@@ -801,7 +809,12 @@ public override object Invoke(XsltContext xsltContext, object[] args, XPathNavig
         // see: http://www.w3.org/TR/xslt#function-function-available
         private class FuncFunctionAvailable : XsltFunctionImpl
         {
+            [RequiresUnreferencedCode(Scripts.ExtensionFunctionCannotBeStaticallyAnalyzed)]
             public FuncFunctionAvailable() : base(1, 1, XPathResultType.Boolean, new XPathResultType[] { XPathResultType.String }) { }
+
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "Suppressing warning here in order to not have to anotate the other 12 subclasses of XsltFunctionImpl " +
+                "which are safe, and we are instead adding an annotation to this subclass constructor")]
```

Comment:
I would just simply say:

```suggestion
            Justification = "The ctor is marked with RequiresUnreferencedCode.")]
```